src/test/java/core/basesyntax/CalculatorImplTest.java

```
@@ -269,35 +270,35 @@ void calculate_raisingMaxPositiveToPositivePower_NotOk() {
 
     @Test
     void calculate_raisingNegativeToPositivePower_Ok() {
-        expected = Math.pow(- 567.042345, 104.4567);
+        expected = NaN;
         actual = negativeAndPositive.calculate('^');
         assertEquals(expected, actual, DELTA);
```

maybe it would be better to throw an exception here because we can't raising to the 'double type' power